openshift/system/Dockerfile

```diff
@@ -52,18 +53,16 @@ ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"
 
 WORKDIR $HOME
 
-RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* \
-    && sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-* \
-    && dnf -y module enable ruby:${RUBY_VERSION} nodejs:16 mysql:8.0 \
-    && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs shared-mime-info postgresql rubygem-irb rubygem-rdoc ruby libpq mysql mysql-libs gd git liberation-sans-fonts file libxml2 libxslt \
+RUN dnf -y module enable ruby:${RUBY_VERSION} nodejs:18 \
+    && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs shared-mime-info postgresql rubygem-irb rubygem-rdoc ruby libpq mysql mariadb-connector-c gd git liberation-sans-fonts file libxml2 libxslt \
     && dnf -y clean all
 
 COPY --from=builder /opt/system/ .
 COPY --from=builder /tmp/memkind-*/jemalloc/lib/libjemalloc.so* /usr/local/lib64
 COPY --from=builder /usr/local/share/gems /usr/local/share/
 COPY --from=builder /usr/local/bin/bundle* /usr/local/bin
 # in RHEL 8 podman-catatonit pulls in too many useless deps so we don't install the RPM directly, on RHEL 9 simply use package `catatonit`
```

Comment:
Apparently, in CentOS 9 it's not the case :grimacing:

Reply:
What do you mean? In CentOS 9 there's a `catatonit` package that only includes the executable: https://centos.pkgs.org/9-stream/centos-appstream-x86_64/catatonit-0.1.7-7.el9.x86_64.rpm.html

Reply:
I think it was giving:

```
$ podman run -it quay.io/centos/centos:stream9 /bin/bash
[root@918ad4e636c6 /]# dnf install catatonit
CentOS Stream 9 - BaseOS                                                                                                                                                                                      2.7 MB/s | 8.2 MB     00:03    
CentOS Stream 9 - AppStream                                                                                                                                                                                   9.0 MB/s |  20 MB     00:02    
CentOS Stream 9 - Extras packages                                                                                                                                                                              17 kB/s |  18 kB     00:01    
Error: 
 Problem: cannot install the best candidate for the job
  - nothing provides podman = 2:4.3.1-3.el9 needed by podman-catatonit-2:4.3.1-3.el9.x86_64 from appstream
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

with `--nobest` it indeed installs `/usr/libexec/catatonit/catatonit`, but skips `podman-catatonit`. Maybe that's OK, as the outcome is the same - we only get a single executable

Reply:
I think it should work. But also this is not so important, so it's fine if you leave it as it is.